masterlist.yaml

```
@@ -7281,6 +7281,8 @@ plugins:
       - 'EconomyOverhaulandSpeechcraftImprovementsLite.esp'
   - name: 'AlchemyAdjustments.esp'
     after: [ 'SkyTEST-RealisticAnimals&Predators.esp' ]
+    tag:
+      - Text
```

Since it's just the one tag, I'd recommend inline formatting as follows:
```suggestion
    tag: [ Text ]
```

---

done